build.gradle

```diff
@@ -50,21 +50,37 @@ subprojects {
       javadoc = true
       tests = false
     }
+    nexus {
+      def gitProvider = new nebula.plugin.info.scm.GitScmProvider()
+      def shortHash = gitProvider.calculateChange(new File('.'))
+      def isRelease = System.getProperty('build.snapshot', 'true') == 'false'
```

Comment:
Please use actual types, we've tried to avoid using def in the rest of gradle/groovy files here.